pkg/ims/models/models.go

```diff
@@ -2,10 +2,14 @@ package models
 
 import "github.com/dgrijalva/jwt-go"
 
-type Users struct {
-	FirstName string `json:"first_name"`
-	LastName  string `json:"last_name"`
+type User struct {
+	UserID int `json:"userID"`
+	FirstName string `json:"firstname"`
+	LastName  string `json:"lastname"`
```

Comment:
JSON tags should be written using a common style, for instance, use camel case or snake case for all tags.